doc/engsys_resource_management.md

```diff
@@ -0,0 +1,107 @@
+## Resource Management Guidelines
+
+This document contains guidelines for creating and managing Azure resources in the Azure SDK Engineering System
+subscriptions. EngSys has automation that will delete resources based on the criteria outlined below. This is to
+improve our overall security and keep us in compliance with company policies.
+
+  * [Managing Dev Resources in the Playground Subscription](#managing-dev-resources-in-the-playground-subscription)
+     * [Resource Groups](#resource-groups)
+     * [Role Assignments](#role-assignments)
+  * [Managing CI resources in the Test Subscription](#managing-ci-resources-in-the-test-subscription)
+     * [Resource Groups](#resource-groups-1)
+     * [Role Assignments](#role-assignments-1)
+
+## Managing Dev Resources in the Playground Subscription
+
+This section applies to resource groups located in the `Azure SDK Developer Playground` subscription.
+
+Currently, the automation will inspect resource groups and role assignments only.
+
+### Resource Groups
+
+The automation will delete all resource groups in the playground subscription DAILY, unless they meet at least one of the following criteria:
+
+- The resource group starts with a valid Microsoft alias within `microsoft.com` or `ntdev.microsoft.com`, followed by an
+  optional hyphen and extra characters.
+    - Valid group name examples: `myalias`, `myalias-test-foobar` for `myalias@microsoft.com`
+- The resource group contains a tag with the name `Owners` where the value is a csv formatted string that contains at
+  least one valid Microsoft alias within `microsoft.com` or `ntdev.microsoft.com`. This convention should only be used
+  when it is not possible to name the resource group with an alias (for example, inner groups auto-created by a resource
+  provider like AKS).
+    - Valid owner tag examples: `Owners: myalias`, `Owners: myalias,anotheralias,lastalias`
+- The resource group contains a tag with the name `DeleteAfter` and an [ISO8601 formatted date value](https://www.iso.org/iso-8601-date-and-time-format.html)
```

Comment:
It should already be more flexible (it just does powershell `-as [DateTime]`, but I don't see much downside to being very specific here. It forces people to generate it rather than typing it in manually, reducing the risk of typos that cause resources to be deleted. Also it makes it easier for us to make changes in the future if we wanted to write or use tooling that wasn't based on the NET `DateTime` type.

Reply:
OK I'm good with being more restrictive in our docs.